### PR TITLE
Added feature request template

### LINKED ISSUE #86: Create Github 'Feature Request' issue template
Last year for Hacktoberfest @Yashesvinee was nice enough to [add an issue template for bug reports](https://github.com/anwilli5/coin-collection-android-US/pull/59). Thanks, Yashesvinee!

That one works great for bugs, but we need one for new feature requests as well. The fields in the bug template don't really make sense for new features, and we'd want to differentiate them with different labels.

More details about Github issue templates:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates

### PR DESCRIPTION
**Pull request by ManasviGoyal**
Fixes #86 
image

### CODE REVIEW

**Comment by ManasviGoyal** [comment]
Please check @sw23

File: .github/ISSUE_TEMPLATE/feature_request_template.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        Kindly search to see if a similar issue [already exists](https://github.com/anwilli5/coin-collection-android-US/issues) for your feature.
+
+        We are also happy to accept contributions from our users. For more details see [here](https://github.com/anwilli5/coin-collection-android-US/blob/master/CONTRIBUTING.md).
+
+  - type: textarea
+    id: feature-description
+    validations:
+      required: true
+    attributes:
+      label: "💭 Description"
```
Comment (sw23) [inline .github/ISSUE_TEMPLATE/feature_request_template.yaml:21]:
```suggestion
      label: "💭 Feature Description"
```

File: .github/ISSUE_TEMPLATE/feature_request_template.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      label: "🧰 Alternatives"
+      description: "Describe alternatives you've considered. A clear and concise description of any alternative solutions or features you've considered."
+
+  - type: textarea
+    id: feature-additional-context
+    validations:
+      required: false
+    attributes:
+      label: "📝 Additional Context"
+      description: "Add any other context about the problem here. Link any related issues here."
```
Comment (sw23) [inline .github/ISSUE_TEMPLATE/feature_request_template.yaml:46]: I think we can keep it simple and just have a single text box
```suggestion
```

**Review by sw23 — CHANGES_REQUESTED** [review]

**Comment by ManasviGoyal** [comment]
Added the requested changes. Please check @sw23. Thanks.

**Review by sw23 — APPROVED** [review]

**Pull request merged**